### PR TITLE
EVG-19012: Pod.events gql field

### PR DESCRIPTION
**Pull request by SupaJoon**
[EVG-19012](https://jira.mongdb.com/browse/EVG-19012)

### Description
The events field will be queried to display container events for the Containers UI

### CODE REVIEW

File: graphql/pod_resolver.go [hunk 1]
```diff
@@ -41,6 +43,28 @@ func (r *podResolver) Task(ctx context.Context, obj *model.APIPod) (*model.APITa
 	return apiTask, nil
 }
 
+func (r *podResolver) Events(ctx context.Context, obj *model.APIPod, limit *int, page *int) (*PodEvents, error) {
+	events, count, err := event.MostRecentPaginatedPodEvents(utility.FromStringPtr(obj.ID), *limit, *page)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("fetching pod events: %s", err.Error()))
+	}
+	// populate eventlogs pointer arrays
```
Comment (Kimchelly) [inline graphql/pod_resolver.go:51]: Do we need this comment? It seems somewhat self-evident that the for loop is building a slice.

File: graphql/pod_resolver.go [hunk 2]
```diff
@@ -41,6 +43,28 @@ func (r *podResolver) Task(ctx context.Context, obj *model.APIPod) (*model.APITa
 	return apiTask, nil
 }
 
+func (r *podResolver) Events(ctx context.Context, obj *model.APIPod, limit *int, page *int) (*PodEvents, error) {
+	events, count, err := event.MostRecentPaginatedPodEvents(utility.FromStringPtr(obj.ID), *limit, *page)
```
Comment (Kimchelly) [inline graphql/pod_resolver.go:47]: Can `limit` or `page` ever be nil, or will GraphQL always ensure they're a non-nil value?

**Review by Kimchelly — CHANGES_REQUESTED** [review]

**Review by khelif96 — APPROVED** [review]

File: graphql/pod_resolver.go (hunk 1 above)
Comment (Kimchelly) [reply graphql/pod_resolver.go:51]: Seems like if this comment doesn't add anything, we should remove it.

**Review by Kimchelly — APPROVED** [review]
LGTM mod follow-up comment.

**Pull request merged**